Mathlib/CategoryTheory/Grothendieck.lean

```diff
@@ -38,11 +39,12 @@ See also `CategoryTheory.Functor.Elements` for the category of elements of funct
 -/
 
 
-universe u
+universe u v u₁ v₁ w
```

Comment:
```suggestion
universe w u v u₁ v₁
```
I think `w` should appear first so that we could do `compAsSmallEquivalenceInverse.{w}` if we need to specify explicit universes for the definitions below.